tests/files/aws/init-services.sh

```diff
@@ -0,0 +1,2 @@
+#!/usr/bin/env bash
+awslocal s3 mb s3://tuf-metadata
```

Comment:
So this is equal to the:
```
awslocal s3api create-bucket --bucket tuf-metadata --region us-east-1
```
command?
What are the differences?
I see we don't provide region here.

What's the difference between `mb` and `create-bucket `?

Reply:
We are using localstack hooks https://docs.localstack.cloud/references/init-hooks/